src/pusher/PusherClient.js

```diff
@@ -0,0 +1,59 @@
+const Pusher = require('pusher-js')
+const batchAuthorizer = require('./batchAuthorizer')
+
+let pusher = null
```

Comment:
I am perpetually confused about where this works and where it doesn't, but I will ask anyways:

could this be a static property on `PusherClient`?

i.e.:
```js
class PusherClient {

  static pusher = null
  ...
}
```

I believe they can be overwritten:
![image](https://user-images.githubusercontent.com/17087405/145191299-c41bca1a-328d-4736-8611-9490f47fb4dc.png)

I don't see it making any difference from how it is now - just a matter of binding state tighter to `PusherClient` instead of global in the file. Just curious.

Reply:
(that image was running node 14.x)

Reply:
edit: I was just doing some of this stuff in hambda
I remember my problem was that the linter would always complain on syntax.
Setting the following in eslint changes that:
`"parserOptions": { "ecmaVersion": 13 },`

Reply:
done